.github/workflows/codeql-analysis.yml

```diff
@@ -3,8 +3,13 @@ name: 'CodeQL'
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - api/*
```

Comment:
Shouldn't this be identical to the ones on pull requests?

Reply:
Yep. I don't know why I did them differently.